modules/core/src/main/scala/org/scalasteward/core/forge/ForgeType.scala

```diff
@@ -41,6 +45,11 @@ sealed trait ForgeType extends Product with Serializable {
   def supportsForking: Boolean = true
   def supportsLabels: Boolean = true
 
+  /** Determines the `head` (GitHub) / `source_branch` (GitLab, Bitbucket) parameter for searching
+    * for already existing pull requests or creating new pull requests.
+    */
+  def pullRequestHeadFor(@nowarn fork: Repo, updateBranch: Branch): String = updateBranch.name
```

Comment:
This is the new home for the `listingBranch()`/`createBranch`  code in the now deleted `modules/core/src/main/scala/org/scalasteward/core/forge/package.scala` file. After correcting `listingBranch()` to output the right string, it was identical to `createBranch`, so there was no point in having two methods- just `pullRequestHeadFor()` will do. The method was originally called `headFor`, back before #649.